Python/pywarpx/picmi.py

```diff
@@ -39,6 +39,7 @@ class constants:
 picmistandard.register_constants(constants)
 
 class Species(picmistandard.PICMI_Species):
+    species_type = None
```

Comment:
The `species_type` attribute doesn't seem to be needed. You could remove it and equivalently check for `self.particle_type == 'photon'`.